spec/hscode/config/bootstrap_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+describe Bootstrap::Config do
+  describe '#get' do
+    let(:root) { File.expand_path('../../../../', __FILE__) }
```

Comment:
Used a top level constant `APP_ROOT`. Then it was stubbed out and replaced with another path to test for `Bootstrap::Config.get`.